apps/zui/docs/features/Time-Display.md

```diff
@@ -0,0 +1,35 @@
+# Time Display
+
+By default, Zui displays Zed `time` values in the Table, Inspector,
+and Detail views using [ZSON format](https://zed.brimdata.io/docs/formats/zson#23-primitive-values).
+This is an [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339)
+date/time string in nanosecond precision ending in `Z` to indicate
+[UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time), e.g.,
+`2024-08-14T19:12:51.123456789Z`.
+
+Starting with Zui release v1.18.0, two different options in Zui **Settings**
```

Comment:
I put in the forward-looking reference to the next Zui release so that way I can merge this right away and ping people on community Slack and/or the person that opened https://github.com/brimdata/brimcap/issues/352 to get early feedback.